### ISSUE TITLE
Plugin is trying to fetch images from CDN that do not exist

### ISSUE THREAD

**Issue by vibonacci** [ealush/emoji-picker-react#233]
![image](https://user-images.githubusercontent.com/9674553/152183597-8b5cbdf9-4913-40aa-9f9b-48c2ae6e2362.png)

This is causing performance issues.

**Comment by bmagnette** [comment]
I got the sames issues.

**Comment by ealush** [comment]
@vibonacci you're right. I completely forgot that I was using a tagged version. Usually you shouldn't use a non-tagged version id, but in our case that's the exact case where we should omit the version, and be fully future compatible.

I just removed the version tag and released 3.5.1. It should now run without these errors (they may still arrive if we face future inconsistencies, but now it should also be self correcting).

Thanks!
